.circleci/config.yml

```diff
@@ -1,12 +1,29 @@
 version: 2.1
 
+orbs:
+  node: circleci/node@4.3.0
+
 jobs:
-  build:
-    docker:
-      - image: circleci/node:12
-        environment:
-          NPM_CONFIG_COLOR: false
+  test:
+    executor:
+      name: node/default
+      tag: <<parameters.version>>
+    parameters:
+      version:
+        type: string
     steps:
       - checkout
-      - run: npm install
+      - run: npm install && git checkout -- package.json
```

Comment:
`npm install` now removes `"dependencies": {}` from __package.json__, causing the `isSortedObject(pkg.dependencies)` assertion to fail. Bad npm. :angry: